spec/index.bs

```diff
@@ -736,13 +736,16 @@ For the duration of each scope, the set of [=internal usage=] flags of any given
 A [=subresource=] may transition to new usages at the boundaries between [=usage scope=]s.
 
 This specification defines the following [=usage scopes=]:
-  1. an individual command on a {{GPUCommandEncoder}}, such as {{GPUCommandEncoder/copyBufferToTexture()|GPUCommandEncoder.copyBufferToTexture}}.
-  2. an individual command on a {{GPUComputePassEncoder}}, such as {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}.
+  1. an individual command on a {{GPUCommandEncoder}}, such as {{GPUCommandEncoder/copyBufferToTexture()}}.
+  2. an individual command on a {{GPUComputePassEncoder}}, such as {{GPUProgrammablePassEncoder/dispatch()}}.
```

Comment:
Can we say `an individual non-state-setting command...`? Maybe say the say for the first bullet too to be consistent.

Reply:
Good clarification.